hal/src/nRF52840/adc_hal.cpp

```diff
@@ -19,8 +19,10 @@
 #include "nrfx_saadc.h"
 #include "adc_hal.h"
 #include "pinmap_impl.h"
+#include "check.h"
 
 static volatile bool m_adc_initiated = false;
```

Comment:
Nit: `state` is probably easier to handle.

Reply:
Fixed.